### DISCUSSION TITLE
Title Buttons (Left and Right)

### DISCUSSION THREAD

**Discussion by madprops** [madprops/grasshopper#166] [General]
I'm adding two buttons to the Main Title, Left and Right buttons.

They appear on mouse hover.

These can be mapped to mouse actions, including showing a menu.

![leftmenu](https://github.com/user-attachments/assets/86ddb085-c058-4f88-aa6e-7c8b1e1479da)

![titlem3n4](https://github.com/user-attachments/assets/37db8351-4202-48f7-a6e7-6924a6d6b39e)

![click34l](https://github.com/user-attachments/assets/9fed5901-76e1-4669-9131-8301e68c411f)

**Comment by N3C2L** [comment]
I love elements that appear on mouse hover, thanks for that!
